### ISSUE TITLE
Empty results; when reading a torch checkpoint, add ability to check for model['state_dict'] ; see below

### ISSUE THREAD

**Issue by wsnoble** [CalculatedContent/WeightWatcher#243]
I am trying to run ww on [this ckpt file](https://github.com/Noble-Lab/casanovo/releases/download/v3.0.0/casanovo_massivekb.ckpt).  Unfortunately, it doesn't seem to produce any results:
```
import torch
import weightwatcher as ww

model = torch.load("casanovo_massivekb.ckpt")
watcher = ww.WeightWatcher(model=model)
results = watcher.analyze()
summary = watcher.get_summary(results)
```

The summary object is `{}` and the results object is

```
Empty DataFrame
Columns: []
Index: []
```

Am I doing something wrong?

**Comment by charlesmartin14** [comment]
**Use this:**

```
import torch
import weightwatcher as ww

model = torch.load("/Users/charleshmartin/Downloads/casanovo_massivekb.ckpt", map_location=torch.device('cpu') )
watcher = ww.WeightWatcher(model=model['state_dict'])
watcher.describe()
```

**Comment by charlesmartin14** [comment]
In the meantime, please upgrade to weightwatcher 0.7.0.9

**Comment by wsnoble** [comment]
Thanks! That worked.

```
watcher.get_summary(results)
{'log_norm': 1.8137791461851493, 'alpha': 3.370289354124059, 'alpha_weighted': 2.5228490603907883, 'log_alpha_norm': 2.978867605640204, 'log_spectral_norm': 0.7530786905225346, 'stable_rank': 14.401966032253991}
```

**Comment by charlesmartin14** [comment]
Let me also suggest trying the new `fix_fingers` option and looking at the details dataframe directly
i.e.
```
details = watcher.analyze(fix_fingers='clip_xmax')
details.alpha.plot.hist()
```


<img width="716" alt="Screenshot 2023-04-05 at 9 21 30 AM" src="https://user-images.githubusercontent.com/498448/230143055-6686668c-a897-46c8-a1ac-b8750300c603.png">

**Comment by charlesmartin14** [comment]
Closing this issue; bug fixed in 0.7.1.1
